2021-offseason/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +14,21 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class kDrivetrain{
+        public static final int frontLeft_ID = 18;
+        public static final int frontRight_ID = 15;
+        public static final int backLeft_ID = 1;
+        public static final int backRight_ID = 14;
+
+        public static final CANSparkMaxLowLevel.MotorType MOTOR_TYPE = CANSparkMaxLowLevel.MotorType.kBrushless;
+
+        public static final int CURRENT_LIMIT = 35;
+        public static final int OPEN_LOOP_RAMP = 0;
+        public static final double SLOW_SPEED = 0.1;
+    }
+    public static final class kOI{//koi fish 
```

Comment:
space between `kOI` and `{`

Reply:
for stuff like open loop ramp having a comment w units would be nice